### ISSUE TITLE
Error with atlas v2.13

### ISSUE THREAD

**Issue by SilasK** [metagenome-atlas/atlas#586]
I tested the latest version on my computer but maybe you have some errors with them. Let's discuss it here.

**Comment by trickovicmatija** [comment]
I had a problem with using minimap2 with indexed reference genomes with Atlas.
The resulting bam files didn't have header, which resulted in problems with samtools afterwards. By looking for help, I found out [here](https://github.com/lh3/minimap2/blob/master/FAQ.md#3-the-output-sam-doesnt-have-a-header) that bam files without headers are created when there is not enough memory while creating index in the previous step.

This can be changed with "-I" flag when creating index. You already thought of that, but named the parameter "memory" instead of "extra" and the wrapper didn't recognized/used it. Therefore, I changed it to "extra" together with changing the format of the memory (-Ixg instead of -IxGb). Now it works perfectly!

Best,
Matija
